example/user/user.proto

```diff
@@ -0,0 +1,73 @@
+syntax = "proto3";
+// Designed to mimic the behavior shown in the GORM example at
+// http://doc.gorm.io/models.html
+
+package user;
+
+import "github.com/infobloxopen/protoc-gen-gorm/options/gorm.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/infobloxopen/protoc-gen-gorm/example/user;user";
+
+message User {
+  option (gorm.opts) = {
+    ormable: true,
+    include: [
+    {type: "int32", name: "billing_address_id"},
+    {type: "int32", name: "shipping_address_id"}
+    ]
+  };
+  int32 id = 1;
+  google.protobuf.Timestamp created_at = 2;
```

Comment:
these can generally be excluded from the proto file, correct? I'm thinking we shouldn't expose them via the API

Reply:
Why not? I could be useful meta data in some cases. Certainly for something like audit log entries...

Reply:
sorry, I should have been more specific: We can include them, but generally they won't need to be included.

I think that in this particular canonical example, we should demonstrate that if you don't specify the `created_at` and `updated_at` fields then they're still included in the database.

so I'm thinking we shouldn't expose them in this example.